src/components/JumbotronVideoComponent.vue

```diff
@@ -27,7 +32,11 @@ if (props.img) {
 </script>
 
 <template>
-  <div ref="mainElementRef" class="jumbotron d-flex align-items-center">
+  <div
+    ref="mainElementRef"
+    :class="isContent ? 'jumbotron-content' : 'jumbotron-title'"
```

Comment:
Variable calculation should be done in <script> rather than <template>. Use a computed property for this! https://vuejs.org/guide/essentials/computed.html